plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -177,6 +177,34 @@ func PodMatchesNodeLabels(pod *api.Pod, node *api.Node) bool {
 	return selector.Matches(labels.Set(node.Labels))
 }
 
+// NoPodLabelConflict evaluates if a pod fits a node by:
+// 1) Comparing the pod's conflict labels against labels of all the other pods.
+// 2) Comparing all the other pods conflict labels with labels of this pod.
+func NoPodLabelConflict(pod *api.Pod, existingPods []*api.Pod, node string) (bool, error) {
+	for podIx := range existingPods {
+		if existingPods[podIx].Labels != nil {
+			for conflict, value := range pod.Spec.Conflicts {
+				if existingValue, ok := existingPods[podIx].Labels[conflict]; ok {
```

Comment:
In the issue I cited, I proposed a list of selectors:
> Another alternative model would be a list of label selectors in the Pod to be used for anti-affinity. 

While individual selectors wouldn't provide top-level OR, the list would: if a pod's labels matched selector1 OR selector2 OR selector3, then it would be considered a conflict.

I don't recommend this mechanism for anti-affinity for like pods, though it could be used that way.